src/transformers/models/fastspeech2/configuration_fastspeech2.py

```diff
@@ -0,0 +1,153 @@
+# coding=utf-8
+# Copyright 2021 The Ontocord team, the G2P, Melgan, and Fastspeech2 Authors, and the HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This software is based on other open source code. A huge thanks to
+# the Huggingface team, and the authors of the Fastspeech2 and Melgan
+# papers and the following authors who originally implemented the
+# various modules, from which this code is based:
+
+# Chung-Ming Chien's Fastspeech2 implementation is under the MIT license: https://github.com/ming024/FastSpeech2
+# Seung-won Park 박승원's Meglan implementation is under BSD-3 license: https://github.com/seungwonpark/melgan
+# Kyubyong Park's G2P implementation is under the Apache 2 license: https://github.com/Kyubyong/g2p, and also here for pytorch specifics https://github.com/Kyubyong/nlp_made_easy/blob/master/PyTorch%20seq2seq%20template%20based%20on%20the%20g2p%20task.ipynb
+""" FastSpeech2 model configuration """
+
+from transformers.configuration_utils import PretrainedConfig
+from transformers.utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+
+import os
+
+class FastSpeech2Config(PretrainedConfig):
+    model_type = "fastspeech2"
+
+    def __init__(self,
+        # Audio and mel
+        ### for LJSpeech ###
+        sampling_rate = 22050,
```

Comment:
Could we remove the space around ` = ` here to comply with our style in other config files? :-)